src/pipeline_blocks.py

```diff
@@ -225,49 +223,53 @@ def classifier_xgb(features, config, train_mode, suffix, **kwargs):
     return xgboost
 
 
-def classifier_sklearn(sklearn_features,
+def classifier_sklearn(features,
                        ClassifierClass,
-                       full_config,
+                       config,
                        clf_name,
                        train_mode,
                        suffix,
-                       normalize,
                        **kwargs):
-    config, model_params, rs_config = full_config
+    model_name = '{}{}'.format(clf_name, suffix)
+    model_params = getattr(config, clf_name)
```

Comment:
@kstrzala Why not simply go with config[clf_name] and config.random_search[clf_name] ?

Reply:
ok, fair